EngineTest/Unit/Features/Fixtures.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace EngineTest.Unit.Features
+{
+    internal class Fixtures
```

Comment:
This `Fixtures` class seems like the more idiomatic way to do things in .NET, rather than the `ConfTest` class I saw above?

Reply:
@asadabbasfolio3 what do you think about this one? Should we rename the `ConfTest` class in `EngineTest.Unit` to `Fixtures` as well?

Reply:
Sorry i missed that one earlier to read , yes `Conftest `and `Fixtures `does the idiomatic behaviour , we should rename the to `Fixtures `as well.